src/histolab/slide.py

```diff
@@ -135,6 +145,53 @@ def level_dimensions(self, level: int = 0) -> Tuple[int, int]:
                 f"{len(self._wsi.level_dimensions)}"
             )
 
+    def level_magnification_factor(self, level: int = 0) -> str:
+        """Return the magnification factor at the specified level.
+
+         Notice that the conversion level-magnification can be computed only
+        if the native magnification is available in the slide metadata.
+
+        Parameters
+        ---------
+        level : int
+            The level which magnification factor is requested, default is 0
+
+        Returns
+        -------
+        magnification factor : str
+            Magnification factor at speficied level
+
+        Raises
+        ------
+        LevelError:
+            If the specified level is not available.
+        KeyError:
+            If the slide's native magnification is not available in the file
+            metadata.
+        """
+        level = level if level >= 0 else self._remap_level(level)
+        if level > (len(self._wsi.level_dimensions) + 1):
+            raise LevelError(
+                f"Level {level} not available. Number of available levels: "
+                f"{len(self._wsi.level_dimensions)}"
+            )
+        properties = self._wsi.properties
```

Comment:
I would use our wrapper
```suggestion
        properties = self.properties
```